curriculum/challenges/english/25-front-end-development/lab-fortune-teller/66c06d618d075c7f7f1b890a.md

```diff
@@ -0,0 +1,153 @@
+---
+id: 66c06d618d075c7f7f1b890a
+title: Build a Fortune Teller App
+challengeType: 14
+dashedName: build-a-fortune-teller-app
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should initialize the five variables `fortune1`, `fortune2`, `fortune3`, `fortune4`, and `fortune5` with a string value of your choice. You can use the below options if you like:
+    
+- `"Your cat will look very cuddly today."`.
+- `"The weather will be nice tomorrow."`.
+- `"Be cautious of your new neighbours."`.
+- `"You will find a new hobby soon."`.
+- `"It would be wise to avoid the colour red today."`.
```

Comment:
```suggestion
- `"Your cat will look very cuddly today."`
- `"The weather will be nice tomorrow."`
- `"Be cautious of your new neighbours."`
- `"You will find a new hobby soon."`
- `"It would be wise to avoid the colour red today."`
```

Pretty minor, but two periods in a row looks weird to me.